pkg/network/tracer/connection/ebpfless/tcp_processor_test.go

```diff
@@ -614,9 +615,8 @@ func TestConnReset(t *testing.T) {
 	require.Equal(t, expectedStats, f.conn.Monotonic)
 }
 
-func TestConnectTwice(t *testing.T) {
-	// same as TestImmediateFin but everything happens twice
-
+func TestProcessResult(t *testing.T) {
```

Comment:
No such thing as connecting twice anymore since the ebpfless tracer will clean up the first connection after close. This test was changed to verify the `ProcessResult` instead